kuma/static/styles/components/contribution-form.scss

```diff
@@ -0,0 +1,272 @@
+                    line-height: 1.75;
+                    font-family: $heading-font-family;
+                }
+
+                h4 {
+                    margin: 15px 0;
+                    font-size: 28px;
+                }
+
+                .button-group a {
+                    color: $text-color;
+                    margin-right: 20px;
+                    min-width: 150px;
+                }
+            }
+        }
+    }
+
+    form {
+        button {
+            height: $input-height;
+            border: none;
+            border-radius: $input-height;
+            line-height: $input-height;
+            background-color: $button-color;
+            font-weight: normal;
+            color: #fff;
+            padding: 0 3em;
+            float: right;
+            @media #{$mq-mobile-and-down} {
+                width: 100%;
+            }
+
+            &:hover {
+                background-color: darken($button-color, 5%);
+            }
+
+            .payment-details {
+                display: none;
+                vertical-align: top;
+                line-height: 0;
+                padding: 10px;
+
+                @media #{$mq-mobile-and-down} {
+                    display: inline-block;
+                }
+            }
+        }
+
+        .form-group {
+            .errorlist {
+                display: block;
+                text-align: right;
+            }
+
+            input {
+                width: 100%;
+                display: block;
+                line-height: 3;
+                box-sizing: border-box;
+                font-weight: normal;
+                padding-left: 12px;
+                border-color: $text-color;
+                margin-top: $input-spacing;
+                &.error {
+                    border-color: $error;
+                    color: $error;
+                }
+            }
+
+            input[name=donation_amount] {
+                padding-left: 35px;
+            }
+
+            .label {
+                position: absolute;
+                margin-left: 20px;
+                margin-top: 40px;
+            }
+
+            input[name=donation_amount],
+            #id_donation_choices {
+                width: 49%;
+                display: inline-block;
+                vertical-align: middle;
+                margin: 0;
+                margin-top: $input-spacing;
+                border-radius: $input-height;
+                @media #{$mq-tablet-and-down} {
+                    display: block;
+                    width: 100%;
+                }
+            }
+        }
+
+        #id_donation_choices {
+            text-align: center;
+
+            li {
+                display: inline-block;
+                position: relative;
+                width: 32%;
+                @media #{$mq-tablet-and-down} {
+                    margin-right: 1em;
+                    width: auto;
+                }
+
+                input[type=radio] {
```

Comment:
Wrap attribute values in single quotes

Reply:
Fixed in https://github.com/mozilla/kuma/pull/4980